src/modules/advancedOrders/state/advancedOrdersAtom.ts

```diff
@@ -26,14 +35,15 @@ export const advancedOrdersAtom = atomWithStorage<ExtendedTradeRawState>(
   createJSONStorage(() => localStorage)
 )
 
-export const updateAdvancedOrdersAtom = atom(null, (get, set, nextState: Partial<ExtendedTradeRawState>) => {
+export const updateAdvancedOrdersAtom = atom(null, (get, set, nextState: Partial<AdvancedOrdersRawState>) => {
   set(advancedOrdersAtom, () => {
     const prevState = get(advancedOrdersAtom)
 
     return { ...prevState, ...nextState }
   })
 })
 
-export const advancedOrdersDerivedStateAtom = atom<TradeDerivedState>({
+export const advancedOrdersDerivedStateAtom = atom<AdvancedOrdersDerivedState>({
   ...DEFAULT_TRADE_DERIVED_STATE,
+  isUnlocked: true,
```

Comment:
Why does it default to true here?

Reply:
I saw that we are doing the same thing [here](https://github.com/cowprotocol/cowswap/blob/58bf46aa0504fcfdeea7cc8439f188c357df7503/src/modules/limitOrders/state/limitOrdersRawStateAtom.ts#L48). One thing I noticed is that it prevents unlock screen flashing when the user already unlocked. But it will work anyways.